captcha.module

```diff
@@ -311,18 +311,20 @@ function captcha_form_alter(array &$form, FormStateInterface $form_state, $form_
       if ($config->get('add_captcha_description')) {
         $captcha_element['#description'] = _captcha_get_description();
       }
+      if ($captcha_point->status()) {
+        // Get placement in form and insert in form.
+        $captcha_placement = _captcha_get_captcha_placement($form_id, $form);
+        _captcha_insert_captcha_element($form, $captcha_placement, $captcha_element);
+      }
```

Comment:
This checks if the captcha is enabled on a form. If it is disabled, it won't place the captcha. (ConfigEntity had the status(), disable() and enable() functionality that does most work for us)